packages/Popover/src/components/Content/Content.js

```diff
@@ -87,8 +88,31 @@ const Content = React.forwardRef((props, ref) => {
     contentStyles.width = content.width;
   }
 
-  /* eslint-disable jsx-a11y/mouse-events-have-key-events */
-  return ReactDOM.createPortal(
+  if (isPortal) {
+    /* eslint-disable jsx-a11y/mouse-events-have-key-events */
+    return ReactDOM.createPortal(
+      <ContentStyled
```

Comment:
should definitely extract this in a variable so that there is no chance of missing one branch of the condition if there is an update to this part